### ISSUE TITLE
Layout analsis fine tuning not producing results as expected and during training loss is not reducing.

### ISSUE THREAD

**Issue by c1ntani** [PaddlePaddle/PaddleOCR#9538]
Hii there ,
I have been trying to fine tune the english fgd model using this pre trained weights https://paddleocr.bj.bcebos.com/ppstructure/models/layout/picodet_lcnet_x1_0_fgd_layout.pdparams

this is my config file
[picodet_lcnet_x1_0_layout.txt](https://github.com/PaddlePaddle/PaddleOCR/files/11064854/picodet_lcnet_x1_0_layout.txt)

this is my coco format json file
[result.txt](https://github.com/PaddlePaddle/PaddleOCR/files/11064857/result.txt)

I have labelled 550 + images using Label-studio and they are working fine.
The main problem that I am facing right now is  when I am training for 300 epochs the loss is not reducing it is stuck between 4 -5.
These are logs which are produced while training.
[Uploading nohup.txt…]()

I have one more doubt

[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37] in pretrained weight head.head_cls0.bias is unmatched with the shape [38] in model head.head_cls0.bias. And the weight head.head_cls0.bias will not be loaded
[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37, 128, 1, 1] in pretrained weight head.head_cls0.weight is unmatched with the shape [38, 128, 1, 1] in model head.head_cls0.weight. And the weight head.head_cls0.weight will not be loaded
[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37] in pretrained weight head.head_cls1.bias is unmatched with the shape [38] in model head.head_cls1.bias. And the weight head.head_cls1.bias will not be loaded
[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37, 128, 1, 1] in pretrained weight head.head_cls1.weight is unmatched with the shape [38, 128, 1, 1] in model head.head_cls1.weight. And the weight head.head_cls1.weight will not be loaded
[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37] in pretrained weight head.head_cls2.bias is unmatched with the shape [38] in model head.head_cls2.bias. And the weight head.head_cls2.bias will not be loaded
[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37, 128, 1, 1] in pretrained weight head.head_cls2.weight is unmatched with the shape [38, 128, 1, 1] in model head.head_cls2.weight. And the weight head.head_cls2.weight will not be loaded
[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37] in pretrained weight head.head_cls3.bias is unmatched with the shape [38] in model head.head_cls3.bias. And the weight head.head_cls3.bias will not be loaded
[03/24 12:47:07] ppdet.utils.checkpoint INFO: The shape [37, 128, 1, 1] in pretrained weight head.head_cls3.weight is unmatched with the shape [38, 128, 1, 1] in model head.head_cls3.weight. And the weight head.head_cls3.weight will not be loaded

This warning comes while training the model.

I need to know which model is more suitable for layout analysis and what should be approach and at the same time for how many images should I train and number of epochs.
Thank you

**Comment by mllife** [comment]
i am also interested in this
